src/python/WMComponent/JobCreator/JobCreatorPoller.py

```diff
@@ -160,6 +164,10 @@ def creatorProcess(work, jobCacheDir):
         swVersion    = work.get('swVersion', None)
         agentNumber  = work.get('agentNumber', 0)
         numberOfCores = work.get('numberOfCores', 1)
+        inputDataset = work.get('inputDataset', None)
+        inputDatasetLocations = work.get('inputDatasetLocations', None)
+        logging.warning("ALEX JobCreator 5: inputDataset %s" % inputDataset)
```

Comment:
I will remove these debugging lines once this is working. I have made this PR to allow Seangchan to easily merge the patches into his testing branch so he can help me debug a problem. I'll open a ticket though ;-)